src/flask-api/net_predictor/WR_logistic_regression.py

```diff
@@ -0,0 +1,119 @@
+import pandas as pd
+from sklearn.linear_model import LogisticRegression
+from sklearn import metrics 
+import kenpom_creds as cred # A file (named "kenpom_creds.py") with proper credentials must be made in the "net-predictor" folder 
```

Comment:
Maybe we should use environment variables since many other projects use them and they would help outsiders understand.  We could even add a dev README for setting up credentials.

https://able.bio/rhett/how-to-set-and-get-environment-variables-in-python--274rgt5#:~:text=To%20set%20and%20get%20environment%20variables%20in%20Python%20you%20can,Get%20environment%20variables%20USER%20%3D%20os.